core/src/autogluon/core/_setup_utils.py

```diff
@@ -18,7 +18,7 @@
     'pandas': '>=1.0.0,<2.0',
     'scikit-learn': '>=0.22.0,<0.25',
     'scipy': '>=1.5.4,<1.7',
-    'gluoncv': '>=0.10.1,<0.12.0',  # FIXME: v0.2 limit version for release
+    'gluoncv': '>=0.10.1.post0,<0.12.0',  # FIXME: v0.2 limit version for release
```

Comment:
For v0.2.0 release, do we need to restrict to exactly `'==0.10.1.post0'`? I don't think older versions will work right?